src/common/rbac.ts

```diff
@@ -87,6 +87,11 @@ export const apiResourceRecord: Record<KubeResource, KubeApiResourceData> = {
     group: "networking.k8s.io",
     namespaced: true,
   },
+  ingressclasses: {
+    kind: "IngressClass",
+    group: "networking.k8s.io",
+    namespaced: true,
```

Comment:
As we found out before, not `namespaced`.

Reply:
Right, fixed.